### ISSUE TITLE
ValueError: numpy.dtype on installation

### ISSUE THREAD

**Issue by vincenzooo** [sfepy/sfepy#313]
[Anaconda2] C:\Users\Vincenzo\sfepy>python setup.py install
Traceback (most recent call last):
  File "setup.py", line 239, in <module>
    check_versions()
  File "setup.py", line 147, in check_versions
    show_only=show_only)
  File "C:\Users\Vincenzo\sfepy\build_helpers.py", line 304, in package_check
    __import__(pkg_name)
  File "C:\Anaconda2\lib\site-packages\tables\__init__.py", line 84, in <module>
    from tables.utilsextension import (
  File "__init__.pxd", line 155, in init tables.utilsextension (tables\utilsextension.c:17736)
ValueError: numpy.dtype has the wrong size, try recompiling

python setup.py build python setup.py build_ext --inplace give same result, I don't know how to "recompile"  so I didn't try.

Windows 10, 
Python 2.7.11 |Anaconda 2.5.0 (64-bit)| (default, Jan 29 2016, 14:26:21) [MSC v.1500 64 bit (AMD64)] on win32Python 2.7.11 |Anaconda 2.5.0 (64-bit)| (default, Jan 29 2016, 14:26:21) [MSC v.1500 64 bit (AMD64)] on win32

**Comment by vincenzooo** [comment]
[Anaconda2] C:\Users\Vincenzo\sfepy>python setup.py build_ext --inplace
[output.txt](https://github.com/sfepy/sfepy/files/175375/output.txt)

**Comment by rc** [comment]
It looks like an issue with pytables in anaconda - try asking there. See also a similar issue [1].

As for the problem in output.txt with `error: Unable to find vcvarsall.bat`, see [2] - it is related to (missing/correct) visual C compilers on windows. Unfortunately, I was not able to solve that problem myself, when trying sfepy in a Windows10 virtualbox image.

[1] https://groups.google.com/forum/?_escaped_fragment_=topic/hddm-users/wWdrpFvBH3c#!topic/hddm-users/wWdrpFvBH3c
[2] https://stackoverflow.com/questions/2817869/error-unable-to-find-vcvarsall-bat#

**Comment by vincenzooo** [comment]
Thanks for the advice, I made some attempt, but I am giving up.
I already had in the past the problem with  vcvarsall.bat on Windows Vista
with some other package and it is a huge mess. 
There are tens of different proposed solutions on the internet that involve
messing up with configuration files, registry etc. I had tried at the time most of them and none had worked.
I will come back to this interesting package if I incidentally solve the problem. 
Thank you
